packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.styles.ts

```diff
@@ -54,76 +54,44 @@ export const inputWrapperBaseStyles = css`
-        color: ${color.light.text.primary.default};
-        background: ${color.light.background.primary.default};
-        border: 1px solid ${color.light.border.primary.default};
-        -webkit-text-fill-color: ${color.light.text.primary.default};
-        box-shadow: ${autofillShadowOverride(
-          color.light.background.primary.default,
-        )};
-
-        &:focus {
-          box-shadow: ${autofillShadowOverride(
-              color.light.background.primary.default,
-            )},
-            ${focusRing.light.input};
-          border-color: ${color.light.border.primary.focus};
-        }
-
-        &:hover:not(:focus) {
-          box-shadow: ${autofillShadowOverride(
-              color.light.background.primary.default,
-            )},
-            ${hoverRing.light.gray};
-        }
-      }
-
-      &::placeholder {
-        color: ${palette.gray.base};
-        font-weight: ${fontWeights.regular};
-      }
-    }
-  `,
+export const getInputWrapperModeStyles = (theme: Theme) => {
+  const isDarkMode = theme === Theme.Dark;
   /** token exceptions: background-color value was designated prior to token system */
-  [Theme.Dark]: css`
-    color: ${color.dark.text.primary.default};
-    background-color: ${palette.gray.dark4};
+  const backgroundColor = isDarkMode
+    ? palette.gray.dark4
+    : color.light.background.primary.default;
+
+  return css`
+    color: ${color[theme].text.primary.default};
+    background: ${backgroundColor};
     border: 1px solid;
 
     & .${inputElementClassName} {
       &:-webkit-autofill {
-        border: 1px solid ${color.dark.border.primary.default};
-        color: ${color.dark.text.primary.default};
-        background: ${palette.gray.dark4};
-        -webkit-text-fill-color: ${color.dark.text.primary.default};
-        box-shadow: ${autofillShadowOverride(palette.gray.dark4)};
+        color: ${color[theme].text.primary.default};
+        background: ${backgroundColor};
+        border: 1px solid ${color[theme].border.primary.default};
+        -webkit-text-fill-color: ${color[theme].text.primary.default};
+        box-shadow: ${autofillShadowOverride(backgroundColor)};
 
         &:focus {
-          box-shadow: ${autofillShadowOverride(palette.gray.dark4)},
-            ${focusRing.dark.input};
-          border-color: ${color.dark.border.primary.focus};
+          box-shadow: ${autofillShadowOverride(backgroundColor)},
+            ${focusRing[theme].input};
+          border-color: ${color[theme].border.primary.focus};
         }
 
         &:hover:not(:focus) {
-          box-shadow: ${autofillShadowOverride(palette.gray.dark4)},
-            ${hoverRing.dark.gray};
+          box-shadow: ${autofillShadowOverride(backgroundColor)},
+            ${hoverRing[theme].gray};
         }
       }
 
       &::placeholder {
-        color: ${palette.gray.dark1};
         font-weight: ${fontWeights.regular};
+        color: color[theme].text.disabled.default;
```

Comment:
hey sorry if my previous comment wasn't clear but can we avoid using the disabled tokens for placeholder? I don't think we should introduce a pattern of doing that since it's a misnomer. can we instead continue using the palette.gray.base and palette.gray.dark1?